README.md

```diff
@@ -207,17 +207,24 @@ Here are the available configurations:
 
 ## Directive
 
-You can use the `[dpDayPicker]` directive to attach the picker to any component with an `ngModel`.
+You can use the `[dpDayPicker]` directive to attach the picker to any component with an `ngModel` or a `FormControl` (using reactive forms).
 
 i.e.
 ```html
-<input name="someName" [(ngModel)]="selectedDate" [dpDayPicker]="config"></input>
+<input name="someName" [(ngModel)]="selectedDate" [dpDayPicker]="config" />
+```
+
+or using reactive forms:
+```html
+<input name="someName" formControlName="selectedDate" [dpDayPicker]="config" />
```

Comment:
why do we need to support this as well? 
this one:  
```
<input name="someName" [formControl]="selectedDateFormControl" [dpDayPicker]="config" />
```
is the official way, right?